### DISCUSSION TITLE
i trying to connect to a strapi user using nuxt auth

### DISCUSSION THREAD

**Discussion by Clemrc** [strapi/strapi#7568] [Legacy Topics]
Hello !

As stated in the title, i trying to connect to a strapi user using nuxt auth but i 'm stuck... I follow this [tutoral](https://www.digitalocean.com/community/tutorials/implementing-authentication-in-nuxtjs-app) but when i want to connect to my strapi api, i got a 404 error with the user endpoint : 

![Capture d’écran 2020-08-23 à 11 38 45](https://user-images.githubusercontent.com/63455928/90975547-6f351a80-e535-11ea-9929-76b2960a9e48.png)

![Capture d’écran 2020-08-23 à 11 38 53](https://user-images.githubusercontent.com/63455928/90975546-6a706680-e535-11ea-8971-233d89830112.png)

![Capture d’écran 2020-08-23 à 11 38 12](https://user-images.githubusercontent.com/63455928/90975528-49a81100-e535-11ea-8d94-a4852b7c69ee.png)

To have acces to the $auth.user.data, we need to call this endpoint.

You can check my repo [here](https://github.com/Clemrc/nuxt_auth) (i only try to connect to a strapi user in this project, nothing else)

Thank's for your help 👍

**Comment by derrickmehaffy** [comment]
Hello, your request URL: `http://localhost:1337/api/auth/user`  I'm not sure where you got that route from? That route doesn't exist within Strapi.

**Reply by Clemrc** [reply]
Yes, this come from the nuxt auth module endpoint, I don't master it well yet. it looks like he's overwriting my url. I still search a solution. i actually look a reddit post about nuxt auth with strapi : https://www.reddit.com/r/Nuxt/comments/gzu9tm/nuxt_auth/
It seems to answer my question a little, but I'm still blocked.

**Reply by derrickmehaffy** [reply]
Yeah it would appear something in your nuxt code isn't sending the request to proper location, you had the correct login endpoint in your first screenshot. Maybe I can ping @Mcastres or @remi2j to see if they can assist you a bit better

**Reply by Mcastres** [reply]
Maybe this guide can help you @Clemrc https://github.com/ramigs/strapi-nuxt-article/blob/master/strapi-nuxt.md

**Reply by Clemrc** [reply]
Oh ! Thank you very much ! i will check 👍
